docs/reference/glossary.md

```diff
@@ -0,0 +1,69 @@
+# Glossaries
+
+## Tools and Technology (general)
+
+```{glossary}
+[Conda](https://docs.conda.io)
+  Package, dependency and environment management for any language—Python, R,
+  Ruby, Lua, Scala, Java, JavaScript, C/ C++, FORTRAN, and more.
+
+[Mamba](https://mamba.readthedocs.io)
+  Is an alternative package manager to conda that is fast, robust, and cross-platform.
+
+[Conda-forge](https://conda-forge.org)
+  Is the main open-access repository for hosting packages that are installed via conda or mamba.
+
+[Docker](https://www.docker.com)
+  Docker provides the ability to package and run an application in a loosely
+  isolated environment called a container. It is widely used for creating
+  reproducible software environments to run code on different computers.
+
+[Git](https://git-scm.com)
+  A popular version control system that is used in many open source software
+  projects to manage their software code base.
+
+[GitHub](https://github.com)
+  Provider of Internet hosting for software development and distributed version
+  control using the "git" command line tool.
+
+[GitHub Actions](https://docs.github.com/en/actions)
+  Continuous integration and continuous delivery (CI/CD) platform that allows you to automate computational workflows for a GitHub respository.
```

Comment:
Suggest saying "feature" instead of "platform" to indicate that his is part of GitHub and not a separate thing.